doc/ref_model/chapters/chapter07.md

```diff
@@ -445,11 +445,11 @@ Ref | Requirement | Definition/Note |
 | Ref | Requirement | Definition/Note |
 |---|----|----|
 | req.sec.wl.001 | The Platform **must** support Workload placement policy | |
-| req.sec.wl.002 | The Platform **must** support operational security | |
+| req.sec.wl.002 | The Cloud Infrastructure Operator **must** provide methods to ensure the platform’s trust status and integrity (e.g. remote attestation, Trusted Platform Module) | |
```

Comment:
I think it should be "The Cloud Infrastructure" instead of "The Cloud Infrastructure Operator". In other words, it is the Infrastructure that provides the method, not the operator

Reply:
@sukhdevkapur Thanks, it's better, done!